lib/languages/sass.js

```diff
@@ -0,0 +1,171 @@
+var util = require('util');
+var DocsParser = require("../docsparser");
+var xregexp = require('../xregexp').XRegExp;
+
+function SassParser(settings) {
+	// this.setup_settings();
+	// call parent constructor
```

Comment:
Other files are indented with 4 spaces, this file is indented with tabs. Could you please change it to tabs too?